Makefile

```diff
@@ -64,6 +64,7 @@ e2e-test: ginkgo ## TODO: Run end to end functional tests.
 ##@ Build
 
 build: container-build ## Build manager binary
+	go build -o ./bin/manager main.go
```

Comment:
Why are you doing a local build *after* the container build? Best I remember, the use of a local go build should not be part of the default build procedure, we should always be using the container build unless the developer has a specific reason they need to do the local build during development.

Beyond this, there's already a `go-build` make target below that uses `./hack/go.build.sh` to run the actual `go build` command.